contracts/eosio.msig/src/eosio.msig.cpp

```diff
@@ -113,6 +132,28 @@ void multisig::unapprove( name proposer, name proposal_name, permission_level le
             a.provided_approvals.erase( itr );
          });
    }
+
+   proposals proptable( get_self(), proposer.value );
+   auto& prop = proptable.get( proposal_name.value, "proposal not found" );
+
+   if ( prop.earliest_exec_time.has_value() ) {
+      if ( prop.earliest_exec_time.value().has_value() ) {
+         auto table_op = [](auto&&, auto&&){};
+         if ( approvals_satisfy_trx_authorization(proposer, proposal_name, prop.packed_transaction, table_op) ) {
+            return;
+         } else {
+            auto prop_it = proptable.find( proposal_name.value );
+            proptable.modify( prop_it, proposer, [&]( auto& p ) {
```

Comment:
There is no need to call `proptable.find` here. You can just pass in the `prop` you retrieved earlier in this function rather than `prop_it`.

Reply:
Previously linked: https://github.com/EOSIO/eosio.contracts/blob/deprecate-deferred3/contracts/eosio.msig/src/eosio.msig.cpp#L145-L155